wcc-contentful-app/app/helpers/wcc/contentful/app/section_helper.rb

```diff
@@ -46,12 +50,66 @@ def markdown(text)
       tables: true
     }
 
-    renderer = ::Redcarpet::Render::HTML.new(options)
+    if links_with_classes.present?
+      renderer = ::WCC::Contentful::App::CustomMarkdownRender.new(options)
```

Comment:
if we properly handled `link_attributes` option, couldn't we just use this always?

Also, the pre & post processing you're doing could be done in the renderer as well? https://github.com/vmg/redcarpet#prepost-process